modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.descriptionPlugin/mh_description.js

```diff
@@ -61,7 +61,12 @@ paella.addPlugin(function() {
       var thisClass = this;
 
       if (thisClass._episode.dcTitle) { this.desc.title = thisClass._episode.dcTitle; }
-      if (thisClass._episode.dcCreator) { this.desc.presenter = thisClass._episode.dcCreator; }
+      if (thisClass._episode.mediapackage.creators) {
+        this.desc.presenter = (Array.isArray(thisClass._episode.mediapackage.creators.creator)
+          ? thisClass._episode.mediapackage.creators.creator
+          : [thisClass._episode.mediapackage.creators.creator])
+          .join(', ');
+      }
       if (thisClass._episode.dcContributor) { this.desc.contributor = thisClass._episode.dcContributor; }
```

Comment:
Not necessary, but it might be nice to also help out multiple contributors while you're at it.
```suggestion
      if (thisClass._episode.dcContributor) { this.desc.contributor = thisClass._episode.dcContributor; }
            if (thisClass._episode.dcContributor) {
        this.desc.contributor = (Array.isArray(thisClass._episode.mediapackage.contributors.contributor)
          ? thisClass._episode.mediapackage.contributors.contributor
          : [thisClass._episode.mediapackage.contributors.contributor])
          .join(', ');
      }
```

Reply:
I think you meant this, right?

```suggestion
      if (thisClass._episode.mediapackage.contributors) {
        this.desc.contributor = (Array.isArray(thisClass._episode.mediapackage.contributors.contributor)
          ? thisClass._episode.mediapackage.contributors.contributor
          : [thisClass._episode.mediapackage.contributors.contributor])
          .join(', ');
      }
```

Reply:
Of course, I meant what you wrote. My apologies.